app/code/Magento/QuoteGraphQl/Model/Resolver/MergeCarts.php

```diff
@@ -7,17 +7,23 @@
 
 namespace Magento\QuoteGraphQl\Model\Resolver;
 
+use Magento\Framework\Exception\CouldNotSaveException;
 use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
 use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
 use Magento\Framework\GraphQl\Query\ResolverInterface;
 use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
-use Magento\QuoteGraphQl\Model\Cart\GetCartForUser;
-use Magento\Quote\Api\CartRepositoryInterface;
 use Magento\GraphQl\Model\Query\ContextInterface;
-use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Model\Cart\CustomerCartResolver;
+use Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface;
+use Magento\QuoteGraphQl\Model\Cart\GetCartForUser;
 
 /**
  * Merge Carts Resolver
+ *
+ * @SuppressWarnings(PHPMD.LongVariable)
```

Comment:
@andrewbess why do we need this?

Reply:
Rest looks good!

Reply:
Hello @prabhuram93 
Because we have a long property in this class `$quoteIdToMaskedQuoteId`